pkgs/so-dang-cool/yn/registry.yaml

```diff
@@ -12,10 +12,18 @@ packages:
       - goos: darwin
         files:
           - name: Yn
+      - goos: linux
+        goarch: arm
```

Comment:
`goarch` must be either amd64 or arm64.

Reply:
Hmm. JSONSchema is wrong.

https://github.com/aquaproj/aqua/blob/13e7eeb8fe05e63add33f1645aeef91e2b154a77/pkg/config/registry/package_info.go#L137-L138

Reply:
Officially aqua supports the following OS and architectures.

https://aquaproj.github.io/docs/products/aqua-registry/contributing#supported-os-and-cpu-architecture

We build aqua for only these platforms.

https://github.com/aquaproj/aqua/blob/13e7eeb8fe05e63add33f1645aeef91e2b154a77/.goreleaser.yml#L16-L22

We can't test and support other platforms.
It's hard to provide environments (laptops and CI environment) for other platforms.
We don't want to make aqua and aqua-registry complicated uselessly.

Reply:
Related issues:

- https://github.com/aquaproj/aqua/issues/1569
- https://github.com/aquaproj/aqua/issues/1758

I looked into how to support other platforms before, though I don't remember well 😅
At least, we need to fix aqua ecosystem to support new platforms.
It's tough, and I don't have the motivation for now.

Reply:
Fixed JSONSchema.

- https://github.com/aquaproj/aqua/pull/3476

Reply:
Thanks for the explanation, and sorry for missing that. Removed arm in 5cf1055a6cec56b1fa7833efed13233bcbacc4d8